youtube_dl/utils.py

```diff
@@ -6604,27 +6604,53 @@ class _UnsafeExtensionError(Exception):
         ),
         # video
         MEDIA_EXTENSIONS.video, (
-            'avif',
```

Comment:
Are these changes not unrelated? Should they happen in a separate PR?

Reply:
Ideally, but they were about to be PRed anyway and I think uncontroversial.